rotkehlchen/chain/aggregator.py

```diff
@@ -1014,18 +1014,28 @@ def _add_eth_protocol_balances(self, eth_balances: DefaultDict[ChecksumEvmAddres
                 addresses=self.queried_addresses_for_module('liquity'),
             )
             for address, staked_info in liquity_staked.items():
-                staked_balance = staked_info['staked'].balance
+                staked_balance = staked_info['balances']['staked'].balance
                 if staked_balance.amount > ZERO:
                     eth_balances[address].assets[A_LQTY] += staked_balance
 
+                for proxy_address, proxy_balance in staked_info['proxies'].items():
+                    staked_balance = proxy_balance['staked'].balance
+                    if staked_balance.amount > ZERO:
+                        eth_balances[proxy_address].assets[A_LQTY] += staked_balance
+
             # Get stability pool balances
             liquity_stability_pool = liquity_module.get_stability_pool_balances(
                 addresses=self.queried_addresses_for_module('liquity'),
             )
             for address, staked_info in liquity_stability_pool.items():
-                staked_balance = staked_info['deposited'].balance
-                if staked_balance.amount > ZERO:
-                    eth_balances[address].assets[A_LUSD] += staked_balance
+                pool_balance = staked_info['balances']['deposited'].balance
+                if pool_balance.amount > ZERO:
+                    eth_balances[address].assets[A_LUSD] += pool_balance
+
+                for proxy_address, proxy_balance in staked_info['proxies'].items():
+                    pool_balance = proxy_balance['deposited'].balance
+                    if pool_balance.amount > ZERO:
+                        eth_balances[proxy_address].assets[A_LUSD] += pool_balance
```

Comment:
Hmmm are you sure this is the correct approach? Here and in line 1024? Afaik, the way we do it with proxies **at the moment** is to add the balances directly to the address that owns the proxy. Otherwise, the way you do it right now, there is gonna be eth balances for an address you don't track and I am not sure this is gonna work with the way the frontend works.

Reply:
While iterating over the PR, I initially went with adding the proxies balances under the owner's address... can't recollect why I changed to this.

Will switch to using the owner's address. Thanks